### DISCUSSION TITLE
Spawning boss and mobs on server side

### DISCUSSION THREAD

**Discussion by jeanpascal92** [Pryaxis/TShock#2158] [Q&A] [answered]
Hello everyone,

Is someone found a way to spawn mobs and boss through the console ?
I've recently tried different plugins such as AutoBoss+ and SpawnMobXY but without success probably because they are not supported in the last version of TShock API.

The idea is to build some custom dungeons and invoke bosses / mobs when a player enters (via SmartRegions).
I also thought about adding a custom account with admin privileges, teleporting to the user, spawning boss then vanishing but this workaround is probably not the best way to do things...

I will be happy to hear about your personal experience on this topic and if you managed to find a solution.

Cheers,
Jean Pascal

**Comment by jeanpascal92** [comment, marked as answer]
Hello, solved by looking at https://github.com/newyrose/SpawnMobX. Exactly does the job for the current tshock version

**Reply by Kojirremer** [reply]
Yep, SpawnMobX is quite a powerful spawner, since it can spawn NPCs at designated coordinates and even alter some of their stats (for instance, an Eye of Cthulhu with a billion HP).
